js/views/modals/BaseModal.js

```diff
@@ -46,9 +46,39 @@ export default class BaseModal extends baseVw {
     };
   }
 
+
+  /**
+   * Returns an array of HTMLElements and if any of these are directly clicked the modal
+   * will be closed. By default, the only target will be this.el, which means any clicks
+   * outside of the modal will close it. But... with the new modal design where everything
+   * is in boxes, the waters are muddied because the overlay is visible inbetween boxes
+   * and you may want a click on those regions to close the modal (e.g. left navigation
+   * is much shorter than the right content exposing a large amount of the overlay).
+   *
+   * Anyhow, since the content inside a modal is very specific to the modal, it is up to
+   * the specific child view to let this base view know what qualifies as a close target.
+   * That should be done by overriding this method. You will very likely want to include
+   * a call to super in the array you are returning, e.g:
+   *
+   *     return [myCustomEl, myCustomEl2, ...super.closeClickTargets];
+   *
+   * Also, the overridden method will be called on EVERY click in this modal, so it is
+   * critical you are caching the elements returned (i.e. DO NOT go to the DOM every time
+   * the method is called).
+   */
+  get closeClickTargets() {
```

Comment:
@tenthhuman Once this code is integrated, you're going to want to override this method in the About modal so clicks on the overlay part of the modal result in the modal being closed. Otherwise, a click on the part under the nav menu wouldn't result in the modal being closed:

![image](https://cloud.githubusercontent.com/assets/794517/21444464/74d9bbec-c862-11e6-8d17-6e6abc3c1319.png)

I did this in the Settings modal, so you could use that as an example. It's virtually the same thing that you'll want to apply to the About modal.